Documentation/applications/interpreters/wamr/index.rst

```diff
@@ -1,3 +1,53 @@
 ==================================
 ``wamr`` Webassembly Micro Runtime
 ==================================
+
+This guides explains how to try WAMR on NuttX.
+
+Install WASI SDK
+================
+
+We need be able to build WASM programs. this can be done by installing the ``wasi-sdk`` as shown in the `WAMR guide <https://wamr.gitbook.io/document/basics/getting-started/host_prerequsites>`_.
+
+Make sure the installation is reachable via ``/opt/wasi-sdk`` on your host, as this is expected by the build script of the WASM hello world sample below.
+
+Building NuttX
+==============
+
+We use the `sim/wamr` config to build WAMR with NuttX. Note that CMake scripts doesn't work, use makefile instead:
+
+.. code:: console
+
+   $ cd nuttx
+   $ tools/configure.sh sim/wamr
+   $ make -j4
+   $ ls -l nuttx
+   -rwxrwxr-x 1 yf yf 4176376 Jun 10 08:11 nuttx
+
+This will generate a ``nutxx`` binary, also a WASM sample will be available in ``apps/interpreters/wamr/wamr/product-mini/app-samples/hello-world`` folder, we build there like below:
```

Comment:
```suggestion
This will generate a ``nuttx`` binary. Also a WASM sample is available in the folder ``apps/interpreters/wamr/wamr/product-mini/app-samples/hello-world``. We build the sample as follows:
```